### PR TITLE
eos_l3interfaces: Add unit testcases to eos_l3_interfaces

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY
Added unit testcases to eos_l3_interfaces

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
test/units/modules/network/eos/fixtures/eos_l3_interfaces_config.cfg

### CODE REVIEW

File: test/units/modules/network/eos/test_eos_l3_interfaces.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+from ansible.modules.network.eos import eos_l3_interfaces
+from units.modules.utils import set_module_args
+from .eos_module import TestEosModule, load_fixture
+
+
+class TestEosL2InterfacesModule(TestEosModule):
+    module = eos_l3_interfaces
+
+    def setUp(self):
+        super(TestEosL2InterfacesModule, self).setUp()
```
Comment (rohitthakur2590) [inline test/units/modules/network/eos/test_eos_l3_interfaces.py:19]: L2  or L3 ?

Comment (Qalthos) [reply test/units/modules/network/eos/test_eos_l3_interfaces.py:19]: Should be L3, yeah

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Comment by Akasurde** [comment]
Hi @GomathiselviS, Thank you very much for your interest in Ansible. This plugin/module is no longer maintained in this repository and has been migrated to https://github.com/ansible-collections/arista.eos<br><br>If you have further questions please stop by IRC or the mailing list:<br><br>* IRC: #ansible on irc.freenode.net<br>* mailing list: https://groups.google.com/forum/#!forum/ansible-project<br><br>needs_info<br>

**Comment by Akasurde** [comment]
Hi @GomathiselviS, Thank you very much for your interest in Ansible. This plugin/module is no longer maintained in this repository and has been migrated to https://github.com/ansible-collections/arista.eos<br><br>If you have further questions please stop by IRC or the mailing list:<br><br>* IRC: #ansible on irc.freenode.net<br>* mailing list: https://groups.google.com/forum/#!forum/ansible-project<br><br>needs_info<br>

**Pull request closed without merging**